### ISSUE TITLE
sun.hlon gives earth longitude w.r.t autumnal equinox and not vernal one

### ISSUE THREAD

**Issue by benjamincensier** [brandon-rhodes/pyephem#67]
when doing something like:

`sun = ephem.Sun()`

`sun.compute(date0)`

`lon_earth = sun.hlon`

to get the longitude of earth in the ecliptic plane, the `lon_earth` result is 0 for a `date0` corresponding to the autumnal equinox. 

That means the longitude is given in a coordinate system where the reference axis is 180° further away from the standard reference axis in an ecliptic system, i.e. the vernal equinox.

So bottom line is: That `sun.hlon` value should be referenced to vernal instead of autumnal equinox
